### ISSUE TITLE
binary bot purchase time

### ISSUE THREAD

**Issue by politecruise** [binary-com/binary-bot#1585]
how do I set a binary.com bot to make the first purchase when the last digit is 8, that appears from the next candlestick, ignoring the current candlestick?

please I need a response A.S.A.P.

**Comment by Hooponopono** [comment]
what means? the bot cannot open a position when the last digit is 8 but will open on the next

**Comment by binary-ex-machina** [comment]
You can't make a purchase on next candle. You can wait for the digit 8 appear and then wait for a new candle to make a purchase.

**Comment by politecruise** [comment]
> what means? the bot cannot open a position when the last digit is 8 but will open on the next


O.k., the idea is that I want to set up a bot that will make a purchase immediately a new candlestick opens, regardless whether its black/red or green/white...

This will make it possible for me to run two or more bots together and they can make a purchase at the same time...

 I have tried different web automation tools like iMacros, but there is still difference in the purchase time in the different bots...

I will appreciate your assistance, thanks.

**Comment by politecruise** [comment]
> You can't make a purchase on next candle. You can wait for the digit 8 appear and then wait for a new candle to make a purchase.

I saw a bot tool  on binary.com site, that says "is candle black", I believe with this it can be manipulated to tell the bot to make a purchase if the answer is true or false, I tried using the tool but still not getting ahead on the issue. 
I will really appreciate your input. Thanks.

![Capture](https://user-images.githubusercontent.com/51704885/59371629-719ac280-8d3d-11e9-94e3-1d9f34234280.PNG)

![Capture](https://user-images.githubusercontent.com/51704885/59371594-5fb91f80-8d3d-11e9-93a0-7c51be5971f9.PNG)
